### ISSUE TITLE
Furigana breaks sentence's parts positioning

### ISSUE THREAD

**Issue by lyfenwebos** [fauu/Kamite#13]
For me, with mpv and enabled furigana, first several kanjis are moved at the end of the sentence when using Edge/Chrome and with Firefox parts are shuffled around when I hover the cursor over the sentence
Issue is 100% reproducible on my end.
Kamite UI > 
![image](https://github.com/fauu/Kamite/assets/25480769/2b3827ad-a6f4-4206-8acd-54812bd22b75)

Subs in the player
![image](https://github.com/fauu/Kamite/assets/25480769/f0b7a287-cd3c-4e65-90d9-9fb9300b5e7b)

**Comment by fauu** [comment]
I wasn't able to reproduce this so far, but maybe I'll come up with of something.

**Comment by lyfenwebos** [comment]
Another behavior that I've encountered is that 1st character is not selectable if are is not in edit mode.
![image](https://github.com/fauu/Kamite/assets/25480769/53f27989-3d3f-46a7-8796-0095cd1da8ba)
![image](https://github.com/fauu/Kamite/assets/25480769/a55c8311-7134-4d46-b7d1-f9f40ce8db55)

**Comment by lyfenwebos** [comment]
Also it is reproduceable only if kamite receives text from mpv socket
When I select the exact same text by manual capture, I can select all characters, even the first one.

Is there any way to increase verbosity for me to get logs for you?

**Comment by fauu** [comment]
No, but the new version will change the relevant code a lot so I'll just finish that up first and see if the problem remains
